### PR TITLE
fix: add download landing page for kam

### PR DESCRIPTION
**Pull request by saumeya**
**What type of PR is this?**
> /kind bug

**What does this PR do / why we need it**:
The current landing page of kam is the default apache page, which doesn't redirect to kam downloads - this PR add the download page and replaces the apache page in the container
 
**Which issue(s) this PR fixes**:

Fixes GITOPS-923

**How to test changes / Special notes to the reviewer**:
Build the image - `docker build --file Dockerfile.serve -t quay.io/<username>/kam .`
Run this image -  `docker run -p 8080:8080 <image>`
On the browser go to - localhost:8080
kam2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!doctype html>
```
Comment (iam-veeramalla) [inline index.html:1]: I would suggest not to have this file in the root directory. This can probably go into a folder like config or something.

Comment (saumeya) [reply index.html:1]: sure, i'll have to create the folder though

Comment (iam-veeramalla) [reply index.html:1]: yes

**Review by iam-veeramalla — APPROVED** [review]
LGTM, Thanks @saumeya

**Comment by iam-veeramalla** [comment]
I think we need @chetan-rns approval to get this merged.

**Review by chetan-rns — APPROVED** [review]
Thanks @saumeya 

/approve

**Pull request merged**